File: dayThree/Java/afternoonSession/MiniParkingSystem.java
public class MiniParkingSystem{

    public static int spaceAvailability(int[] slot, int slotNumber, String response){
    
        int availableSpace = 0;
        
          
        for(int index = 0; index < slot.length; index++){
        
            if (slot[index] == 0){
                slot[index] = 1;
                break; 
            }
        }
        
        if (response.equals("yes"))
            
            slot[slotNumber-1] = 0;
            
        
        for(int index = 0; index < slot.length; index++){
            
            if (slot[index] == 0)
                availableSpace += 1;
            
        }return availableSpace;
        
        
    
    }

    public static int spaceFilled(int[] slot, int slotNumber, String response){
    
        int space = spaceAvailability(slot, slotNumber, response);
        int filledSpace = slot.length - space;
        return filledSpace;
        
    
    }


}

File: dayThree/Python/afternoonSession/mini_parking_system.py
def space_availability(slot, slot_number, response):
    
    available_space = 0

      
    for index in range(len(slot)):

        if slot[index] == 0:
            slot[index] = 1
            break

    if response == "yes":
        
        slot[slot_number-1] = 0
        

    for index in range(len(slot)):
        
        if slot[index] == 0:
            available_space += 1
        
    return available_space
        
        
    
def space_filled(slot, slot_number, response):
    
    space = space_availability(slot, slot_number, response)
    filled_space = len(slot) - space
    return filled_space
        
    
    



